samples/contacts_sample.py

```diff
@@ -74,41 +58,29 @@ def add_a_contact_to_a_contact_list():
         "ContactID": "987654321",
         "ContactAlt": "passenger@mailjet.com",
         "ListID": "123456",
-        "ListAlt": "abcdef123"
+        "ListAlt": "abcdef123",
     }
     return mailjet30.listrecipient.create(data=data)
 
 
 def manage_the_subscription_status_of_an_existing_contact():
     """POST https://api.mailjet.com/v3/REST/contact/$contact_ID
-                /managecontactslists"""
+    /managecontactslists"""
     _id = "$contact_ID"
     data = {
         "ContactsLists": [
-            {
-                "Action": "addforce",
-                "ListID": "987654321"
-            },
-            {
-                "Action": "addnoforce",
-                "ListID": "987654321"
-            },
-            {
-                "Action": "remove",
-                "ListID": "987654321"
-            },
-            {
-                "Action": "unsub",
-                "ListID": "987654321"
-            }
+            {"Action": "addforce", "ListID": "987654321"},
+            {"Action": "addnoforce", "ListID": "987654321"},
+            {"Action": "remove", "ListID": "987654321"},
+            {"Action": "unsub", "ListID": "987654321"},
         ]
     }
     return mailjet30.contact_managecontactslists.create(id=_id, data=data)
 
 
 def manage_multiple_contacts_in_a_list():
     """POST https://api.mailjet.com/v3/REST/contactslist/$list_ID
-                /managemanycontacts"""
+    /managemanycontacts"""
```

Comment:
I'm not sure if it's worth to split this URL, especially if it's shorter then 79 characters

Reply:
Fair enough!

Reply:
Fixed